exercises/resistor-color/lib/example.dart

```diff
@@ -0,0 +1,5 @@
+class ResistorColor {
+  final colors = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"];
```

Comment:
You might consider moving this to the package level and making it private.

Reply:
One or the other works for me. Or both. :)

Reply:
@jvarness It looks like it needs to be public in order to fit pass a test.